css/style.css

```diff
@@ -1,9 +1,213 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
-    font-family: 'Roboto', sans-serif;
+ 
+ body {
+    font-family: 'Roboto',
+    sans-serif;
     -webkit-font-smoothing: antialiased;
+    background-color: white;
+     }
+/*General styles*/
+
+/*Navigation styles*/
+
+#header {
+    background-color: white;
+    border-bottom: 1px solid grey;
+    top: 0;
+    width: 100%;
+}
+
+#logo {
+    width: 30px;
+    padding-left: 200px;
+    float: left;
```

Comment:
Can you replace `float: left` with flexbox?
It is not recommended to use floats, they were used before flexbox was created, and are now deprecated.